### DISCUSSION TITLE
Maintenance branches for monorepo

### DISCUSSION THREAD

**Discussion by codekult** [semantic-release/semantic-release#3526] [Configuration] [answered]
Hi, I need help configuring maintenance branches for a monorepo. I know that this question exceed the purpose of semantic-release library but it won't hurt to ask for some advice here.

We're using [semantic-release-monorepo](https://github.com/pmowrer/semantic-release-monorepo) to integrate semantic-release with our multiple packages. Releases are generated following a pattern like `@orgName/packageName-v1.0.0`.

I would like to be able to create maintenance branches following the same pattern, ex: `packageName-1.x` or `packageName-1.2.x`, etc.

**Comment by travi** [comment, marked as answer]
We do not provide support for semantic-release-monorepo here. Please ask for support there about your monorepo related questions. Tags and branches impact an entire repository. The namespacing that you mention is specific to semantic-release-monorepo and is not a consideration in the functionality we support here
